src/pods/canvas/canvas.pod.tsx

```diff
@@ -18,8 +18,15 @@ import {
 import { CanvasSvgComponent } from './canvas-svg.component';
 import { EditRelationPod } from '../edit-relation';
 import { mFlix } from './m-flix.mock.data';
+import useAutosave from '@/core/autosave/autosave.hook';
 
 export const CanvasPod: React.FC = () => {
+  const { stopAutosave } = useAutosave();
```

Comment:
```ts
const { startAutosave, stopAutoSave} = useAutosave();
```

By doing this we have granular control when we start  (component mounted) and stop